### PR TITLE
feat: add translations

### LINKED ISSUE #11: [Feature request] add 🈯 translations
### 🈯 Translations
Add the possibilty to load content from language files for:

- German
- English

and of cause be a place for all upcoming languages

### PR DESCRIPTION
**Pull request by xXjojaXx**
closes #11

### CODE REVIEW

File: src/components/layout/Sidebar.svelte [hunk 1]
```diff
@@ -10,36 +10,39 @@
 		BookOpenIcon,
 		SettingsIcon,
 	} from 'svelte-feather-icons';
+	import { link } from 'svelte-spa-router';
```
Comment (tametsi) [inline src/components/layout/Sidebar.svelte]: remove unused imports from `svelte-spa-router`

File: src/components/pages/Analytics.svelte [hunk 2]
```diff
@@ -14,7 +16,9 @@
 		? selectedRunner
 		: $activeSession.run.runners[0];
 	$: data = {
-		labels: selectedRunner?.rounds.all.map((_, i) => `Round: ${i + 1}`),
+		labels: selectedRunner?.rounds.all.map(
+			(_v, i) => `${get(_)('analytics.round')}: ${i + 1}`
```
Comment (tametsi) [inline src/components/pages/Analytics.svelte]: use `$` instead of `get`

File: src/components/pages/Edit.svelte [hunk 3]
```diff
@@ -21,49 +22,50 @@
...
-					data.
+					{$_('edit.alias_description')}
 				</svelte:fragment>
 			</FormItem>
 
 			<div class="py-2">
-				<button type="submit" class="btn btn-primary">Add Runner</button
-				>
+				<button type="submit" class="btn btn-primary"
+					>{$_('edit.add_runner')}
```
Comment (tametsi) [inline src/components/pages/Edit.svelte]: style: whitespace consistency

File: src/lang/de.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+			max_difference: 'Max. Unterschied',
+		},
+	},
+	about: {
+		title: 'Über LIGAS',
+		app_title: 'LIGAS - Zeit messen leicht gemacht!',
+		app_description: [
+			'LIGAS ist ein Werkzeug, um das Messen',
+			'und die Verwaltung',
+			'von Rundenzeiten während einem Wettkampf zu erleichtern.',
```
Comment (tametsi) [inline src/lang/de.ts]: der dativ ist dem genitiv sein tod

File: src/lib/run.ts [hunk 5]
```diff
@@ -14,12 +16,12 @@ export default class Run {
...
-			'Rounds',
-			'Distance',
-			'Max. Difference (in s)',
-			'Rounds (Format: mm:ss:ff)',
+			get(_)('runner.stats.name'),
+			get(_)('runner.stats.alias'),
+			get(_)('runner.stats.rounds'),
+			get(_)('runner.stats.distance'),
+			`${get(_)('runner.stats.max_difference')} (in s)`,
+			`${get(_)('runner.stats.rounds')} (Format: mm:ss:ff)`,
```
Comment (tametsi) [inline src/lib/run.ts]: looks like `in`and `Format` are international words now...

File: src/components/elements/HistoryEntry.svelte [hunk 6]
```diff
@@ -76,21 +84,21 @@
 				class="btn btn-sm"
 				title="Loads this session. This session will be modified."
```
Comment (tametsi) [inline src/components/elements/HistoryEntry.svelte]: missing translations for `title`

**Review by tametsi — CHANGES_REQUESTED** [review]
pls resolve the comments I left  
and add translations for this line: https://github.com/tametsi/ligas/blob/46e3649bda7b44fa6d419ffa53f49b9116c03858/src/components/layout/Navbar.svelte#L15

**Comment by xXjojaXx** [comment]
All comments of the review should be resolved by the two commits above.
Please inspect the new changes again before merging the pull request.
享受翻译的乐趣！

**Review by tametsi — APPROVED** [review]

**Pull request merged**